articles/micro_transactions_and_the_underlinin_values.md

```diff
@@ -50,6 +50,83 @@ I’m currently thinking of doing: Solana, Polygon, Dash, Nano, Litecoin, Bitcoi
+Now let's look at the computer power needed to run a one of these networks when they are operating at scale. Fortunately, there are two platforms already considered to be at this level, which are Solana and Polygon.
+
+## Solana’s Computing power
+
+Solana is a highly scalable layer 1 Blockchain built for mass adoption. The requirements to run a validator node are generally unreachable for the typical user, and starts at needing:
+* A high-speed internet connection
+* A CPU with 12 cores / 24 threads, or more
+* 256 GB of RAM or more, the recommendation is 512 GB of RAM
+* 500 GB, or larger Disk space for accounts
+* 1 TB or larger disk space for Ledger
+
+As per Solana’s specifications, the specs listed here should be considered the bare minimum — https://docs.solanalabs.com/operations/requirements
+
+Solana produces approximately 1 TB of data per day. To keep the ledge size manageable, the platform utilises pruning and sharding methods. 
+
+Even with all this capacity, it’s still worth noting that on both Solana and Polygon, you can’t send a micro transaction of a single unit for a single unit fee.
+
+## Decred’s LN Computing power 
+
+In comparison, Decred’s Lightning Network is run using the platform's Lightweight SPV mode. Which requires less than 2 GB of storage, 1 GB of RAM and can operate seamlessly over an internet connection of approximately 2 MB per second.
+
+Over the lightning Network, you can send a single unit at the cost of a single unit (0.00000001 DCR). Which is also the standard transaction fee for transactions of 1 KB of data or less.
+
+The Lightning Network conducts the majority of its transactions off-chain. Only the opening and closing of payment channels are recorded on-chain. This allows for greater scalability at minimum cost in all six areas listed previously. Decred’s LN is also fully non-custodial and extremely secure.
+
+The Lightning Network gets a lot of criticism in the crypto space. But as I continue to conduct my research, this seems more to do with envy and the fact that others can’t achieve the same levels of transaction volume, without considerable trade-offs. 
+
+When implemented properly, as it is on the Decred blockchain, the Lightning Network is currently the only crypto platform capably of maintaining peer to peer payment functionality at a scale of 100,000 TPS. The Lightning Network is already capable of processing similar qualities of transactions per second to that of Visa’s payment network. Which is impressive when you consider this is still a relatively young technology.
```

Comment:
maybe you could put in the end a little step by step or info on how to use the DCR lightning network

Reply:
Good point but not really in the scope of this article. As a side note, and in my opinion, the best way to use LN on Decred is through Bison Relay which I created the step by step setup in my Bison Relay video series. 

Using LN in Decrediton is pretty difficult to setup and get started but Bison Relay is very simple.